actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -67,8 +67,9 @@ def p(*arguments, **options, &block)
 
         def tag_string(name, content = nil, escape_attributes: true, **options, &block)
           content = @view_context.capture(self, &block) if block_given?
-          if (HTML_VOID_ELEMENTS.include?(name) || SVG_VOID_ELEMENTS.include?(name)) && content.nil?
-            "<#{name.to_s.dasherize}#{tag_options(options, escape_attributes)}>".html_safe
+          self_closing = SVG_SELF_CLOSING_ELEMENTS.include?(name)
```

Comment:
Why is this now a local variable? It changes the order of execution, which can make this method slower

Reply:
Good point. But it would only be slower for the void element calls. I was trying to avoid the check again when determining the self closing. Should I make a separate elsif clause instead?